tests/UI/utils/keycloakHelper.ts

```diff
@@ -0,0 +1,88 @@
+import {APIRequestContext, APIResponse, Page} from 'playwright';
+
+/**
+ * @description Helper to use Keycloak
+ */
+export default {
+
+  async login(page: Page, username: string, password: string): Promise<void> {
```

Comment:
I think the JS doc is missing and maybe it's better to create variables for selectors wdyt ?

Reply:
Done